README.md

```diff
@@ -7,12 +7,12 @@ Spoke is an open source text-distribution tool for organizations to mobilize sup
 
 Spoke was created by Saikat Chakrabarti and Sheena Pakanati, and is now maintained by MoveOn.org.
 
-The latest version is [5.4](https://github.com/MoveOnOrg/Spoke/tree/v5.4) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/main/docs/RELEASE_NOTES.md#v54))
+The latest version is [5.4](https://github.com/MoveOnOrg/Spoke/tree/v5.5) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/main/docs/RELEASE_NOTES.md#v55))
```

Comment:
the link text needs to be 5.5

Reply:
Wow. This is such a blindspot of mine. Thank you!